packages/template-cli/package.json

```diff
@@ -15,9 +15,10 @@
     "src/domain"
   ],
   "scripts": {
+    "license:check:audit": "license-checker --production --out ./packageLicensesAudit.json --excludePrivatePackages --relativeLicensePath",
+    "license:check:allowed": "license-checker --production --excludePrivatePackages --onlyAllow='MIT;MIT*;ISC;BSD-3-Clause;BSD-2-Clause;Apache-2.0;Apache*;CC-BY-4.0;WTFPL'",
     "test": "node_modules/.bin/jest -c jest.config.json --coverage",
     "test:watch": "node_modules/.bin/jest -c jest.config.json --coverage --watchAll",
-    "lint": "node_modules/.bin/eslint . -f codeframe",
```

Comment:
ah wait this is part of the root right?

Reply:
Good shout, can remove this. No linting here.